backend/wayland/wl_seat.c

```diff
@@ -56,6 +56,9 @@ static void pointer_handle_leave(void *data, struct wl_pointer *wl_pointer,
 	}
 
 	struct wlr_wl_output *output = wl_surface_get_user_data(surface);
+	if (output == NULL) {
+		return;
+	}
```

Comment:
This can be an assert

Reply:
Right-o, changed in pointer_handle_enter as well. Force pushed along with braces fix in their appropriate commits